src/app/zap-templates/zcl/data-model/chip/general-diagnostics-cluster.xml

```diff
@@ -89,5 +89,24 @@ limitations under the License.
     <attribute side="server" code="0x05" define="ACTIVE_HARDWARE_FAULTS" type="ARRAY" entryType="ENUM8" length="254" writable="false" optional="true">ActiveHardwareFaults</attribute>
     <attribute side="server" code="0x06" define="ACTIVE_RADIO_FAULTS" type="ARRAY" entryType="ENUM8" length="254" writable="false" optional="true">ActiveRadioFaults</attribute>
     <attribute side="server" code="0x07" define="ACTIVE_NETWORK_FAULTS" type="ARRAY" entryType="ENUM8" length="254" writable="false" optional="true">ActiveNetworkFaults</attribute>
+    <event side="server" code="0x00" name="HardwareFaultChangeEvent" priority="critical" optional="true">
```

Comment:
Why the "Event" appended to every event name?  This is going to end up being something like `Events::HardwareFaultChangeEvent`, which is pretty redundant.  Why not just use the spec names?